docs/docs/snapshots.md

```diff
@@ -6,14 +6,13 @@ title: Database Snapshots
 
 You can download a snapshot of the Juno database to reduce the network syncing time. Only the blocks created after the snapshot will be synced when you run the node. Fresh snapshots are automatically uploaded once a week and are available under the links below.
 
-Snapshots are provided in a compressed `.tar.zst` format for faster downloads and reduced storage requirements. It also allows you to directly stream the decompressed file to your computer without needing to download it first. 
+Snapshots are provided in a compressed `.tar.zst` format for faster downloads and reduced storage requirements. It also allows you to directly stream the decompressed file to your computer without needing to download it first.
 
-
-| Network | Version | Download Link |
-|-------- | ------- | ------------- |
-| Mainet | **>=v0.13.0**  | [**juno_mainnet.tar.zst**](https://juno-snapshots.nethermind.io/files/mainnet/latest) |
-| Sepolia | **>=v0.13.0** | [**juno_sepolia.tar.zst**](https://juno-snapshots.nethermind.io/files/sepolia/latest) |
-| Sepolia-Integration | **>=v0.13.0** | [**juno_sepolia.tar.zst**](https://juno-snapshots.nethermind.io/files/sepolia/latest) |
+| Network             | Version       | Download Link                                                                                                 |
+| ------------------- | ------------- | ------------------------------------------------------------------------------------------------------------- |
```

Comment:
my md-formatter did a strange thing, but based on the previous formatting and the how it look when rendered, I think it's fine?

https://juno.nethermind.io/snapshots/

Reply:
I've tested it locally, and it looks fine :)